code/game/objects/items/kitchen.dm

```diff
@@ -146,15 +146,15 @@
 	throwforce = 15
 	throw_speed = 5
 	throw_range = 7
-	var/amount_per_transfer_from_this = 5
+	var/amount_per_transfer_from_this = 20
```

Comment:
20 is too much in my opinion. I'd reduce TC cost further and keep it at 10.
20 means you would transfer the entire amount in just 2 hits, which is too quick.